lua/dap.lua

```diff
@@ -1060,16 +1059,17 @@ function Session:_step(step, params)
   end)
 end
 
-function M.step_over()
+function M.step_over(options)
   if not session then return end
-  session:_step('next')
+  session:_step('next', options)
 end
 
-function M.step_into()
+function M.step_into(options)
   if not session then return end
-  local ask_targets = M.defaults[session.config.type].ask_step_in_targets
-  if not (ask_targets and session.capabilities.supportsStepInTargetsRequest) then
-    session:_step('stepIn')
+  local askForTargets = options.askForTargets
```

Comment:
```suggestion
  local askForTargets = options and options.askForTargets
```

Otherwise this causes an error if options is nil